test/tests/interfacekernels/InterfaceSorption/tests

```diff
@@ -319,11 +334,26 @@
                 InterfaceKernels/interface/flux_penalty=1e1
                 Outputs/file_base=interface_sorption_transient_Henry_non_ad_penalty_scaling_out'
     exodiff = 'interface_sorption_transient_Henry_non_ad_penalty_scaling_out.e'
+    mesh_mode = REPLICATED
+    recover = false # this is discussed in #196
+  []
+  [Henry_transient_non_ad_penalty_scaling_comparison]
+    requirement = 'The system shall have the capability to enforce interfacial conditions based on the Henry law in isothermal conditions using a penalty-enforced flux balance during transient simulations with unit scaling on both variables and provide similar results to the approach without the penalty-enforced flux balance.'
+    type = 'Exodiff'
+    input = 'interface_sorption_transient.i'
+    cli_args = 'n_sorption=1
+                unit_scale=4
+                unit_scale_neighbor=6
+                InterfaceKernels/interface/use_flux_penalty=true
+                InterfaceKernels/interface/flux_penalty=1e1
+                Outputs/file_base=interface_sorption_transient_Henry_non_ad_penalty_scaling_comparison_out'
+    should_execute = false # this test relies on the output files from Henry_transient_non_ad_penalty_scaling, so it shouldn't be run twice
```

Comment:
This line seems to be causing the test to fail, as no exodus file for comparison is being created to compare against.

Reply:
Hmmm, it was working on my laptop and I was surprised that is was. But I must have had the output already present.

Reply:
Done